FRAMEWORKS.yml

```diff
@@ -191,6 +191,10 @@ go:
     website: godoc.org/github.com/kataras/muxie
     version: "1.0"
     language: "1.11"
+  gf:
+    website: gfer.me
+    version: "1.1.994"
```

Comment:
could you use the format : X.Y -> it will be more readable for our audience :+1:

Reply:
ok, done